### PR TITLE
Remove ethereum-address dependency and add this logic to the repo

### PR DESCRIPTION
**Pull request by LogvinovLeon**
This PR:
* Resolves: #254

### CODE REVIEW

**Comment by fabioberger** [comment]
Can we please wait for #253 to be merged and then move this to the `utils` subpackage and use it throughout all the 3 subpackages that rely on this dependency?

File: packages/assert/package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "url": "https://github.com/0xProject/0x.js/issues"
   },
   "homepage": "https://github.com/0xProject/0x.js/packages/assert/README.md",
+  "dependencies": {
```
Comment (BMillman19) [inline packages/assert/package.json]: merge this and the other dependencies struct

File: packages/assert/package.json [hunk 2]
```diff
@@ -39,7 +42,7 @@
   "dependencies": {
     "@0xproject/json-schemas": "^0.6.10",
     "bignumber.js": "~4.1.0",
-    "ethereum-address": "^0.0.4",
+    "js-sha3": "^0.7.0",
```
Comment (BMillman19) [inline packages/assert/package.json]: shouldnt this go in the utils package dependencies?

File: packages/website/ts/components/inputs/address_input.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {isAddress} from 'ethereum-address';
+import {addressUtils} from '@0xproject/utils';
```
Comment (BMillman19) [inline packages/website/ts/components/inputs/address_input.tsx]: we need a corresponding change on line 71

**Pull request merged**